doc/API.md

```diff
@@ -32,13 +32,17 @@ Lists all instances which have been provisioned by Service Maker. Can be queried
 
 ### `POST` /v1/instances
 Creates a new instance of the specified type.
-
+* If neither createSecurityGroup nor existingSecurityGroup are specified, the default security group is used.
+* If neither createKeyName nor existingKeyName are specified, the default key name is used.
 #### Payload format
```

Comment:
We should probably specify the headers with this payload, especially since we use a custom one.